geoportal/c2cgeoportal_geoportal/scripts/theme2fts.py

```diff
@@ -289,3 +289,53 @@ def _add_layer(
             self._add_fts(layer, interface, "add_layer", role)
 
         return fill
+
+    def _render_label(
+        self,
+        item: "c2cgeoportal_commons.models.main.TreeItem",
+        lang: str,
+    ) -> Optional[str]:
+        patterns = item.get_metadata("searchLabelPattern")
+        if not patterns:
+            return None
+        pattern = patterns[0]
+        tree_paths = list(self._get_paths(item))
+        # Remove paths where the last element isn't a theme
+        tree_paths = [p for p in tree_paths if p[-1].item_type == "theme"]
+        result = None
+        if tree_paths:
+            for path in tree_paths:
+                if len(path) == 2:
+                    current_result = str(pattern.value).format(
```

Comment:
`pattern.value` is not already a string?

Reply:
It should already be a string but if I remove it on the return line, I get a type error, let me know if you have a better way to handle this https://github.com/camptocamp/c2cgeoportal/blob/9abff086881e40768afbfb599c195957fa6af500/geoportal/c2cgeoportal_geoportal/scripts/theme2fts.py#L328